### PR TITLE
Ml commons

### PR DESCRIPTION
**Pull request by dtaivpp**
### Description
This PR does the following: 

1. Cleans up ML index page (related to #4869)
    - Previously the index page contained cluster settings and model permissions needed. These have been moved to the appropriate sections and are now referenced from the top page. 
2. Fixed code sample issue where incorrect sample was used for external connectors.
3. Updated ML Cluster settings and ML Model Access pages to have appropriate data previously found in index page. 

### Issues Resolved
Closes https://github.com/opensearch-project/ml-commons/issues/1332

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _ml-commons-plugin/cluster-settings.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ nav_order: 160
 
 To enhance and customize your OpenSearch cluster for machine learning (ML), you can add and modify several configuration settings for the ML Commons plugin in your 'opensearch.yml' file.
 
+## ML node
+
+By default ML tasks and models only run on ML nodes. When configuring without the `data` node role, ML nodes will not store any shards and will calculate resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Give your node a custom name and define the node role as `ml`:
```
Comment (kolchfa-aws) [inline _ml-commons-plugin/cluster-settings.md:14]:
```suggestion
By default, ML tasks and models only run on ML nodes. When configured without the `data` node role, ML nodes do not store any shards and instead calculate resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Give your node a custom name and define the node role as `ml`:
```

File: _ml-commons-plugin/index.md [hunk 2]
```diff
@@ -17,18 +17,10 @@ Models [trained]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#training-the
 
 Should you not want to use a model, you can use the [Train and Predict]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#train-and-predict) API to test your model without having to evaluate the model's performance.
 
-# Permissions
+## Utilizing ML Commons
```
Comment (kolchfa-aws) [inline _ml-commons-plugin/index.md:20]:
```suggestion
## Using ML Commons
```

File: _ml-commons-plugin/index.md [hunk 3]
```diff
@@ -17,18 +17,10 @@ Models [trained]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#training-the
...
-
-To prevent your cluster from failing when running ML tasks, you configure a node with the `ml` node role. When configuring without the `data` node role, ML nodes will not store any shards and will calculate resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Give your node a custom name and define the node role as `ml`:
-
-```yml
-node.name: ml-node
-node.roles: [ ml ]
-```
+1. Ensure you've set the appropriate cluster settings outlined in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/). 
+2. Set up model access described in [Model Access Control]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control/). 
+3. Start utilizing models:
```
Comment (kolchfa-aws) [inline _ml-commons-plugin/index.md:24]:
```suggestion
3. Start using models: 
```

File: _ml-commons-plugin/index.md [hunk 4]
```diff
@@ -17,18 +17,10 @@ Models [trained]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#training-the
...
-To prevent your cluster from failing when running ML tasks, you configure a node with the `ml` node role. When configuring without the `data` node role, ML nodes will not store any shards and will calculate resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Give your node a custom name and define the node role as `ml`:
-
-```yml
-node.name: ml-node
-node.roles: [ ml ]
-```
+1. Ensure you've set the appropriate cluster settings outlined in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/). 
+2. Set up model access described in [Model Access Control]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control/). 
+3. Start utilizing models: 
+  - [ML Framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/ml-framework/) for running models within OpenSearch.
```
Comment (kolchfa-aws) [inline _ml-commons-plugin/index.md:25]:
```suggestion
  - [ML Framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/ml-framework/) allows you to run models within OpenSearch. 
```

File: _ml-commons-plugin/index.md [hunk 5]
```diff
@@ -17,18 +17,10 @@ Models [trained]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#training-the
...
-
-```yml
-node.name: ml-node
-node.roles: [ ml ]
-```
+1. Ensure you've set the appropriate cluster settings outlined in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/). 
+2. Set up model access described in [Model Access Control]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control/). 
+3. Start utilizing models: 
+  - [ML Framework]({{site.url}}{{site.baseurl}}/ml-commons-plugin/ml-framework/) for running models within OpenSearch. 
+  - [ML Extensibility]({{site.url}}{{site.baseurl}}/ml-commons-plugin/extensibility/index/) to access remote models.
```
Comment (kolchfa-aws) [inline _ml-commons-plugin/index.md:26]:
```suggestion
  - [ML Extensibility]({{site.url}}{{site.baseurl}}/ml-commons-plugin/extensibility/index/) allows you to access remote models. 
```

**Review by kolchfa-aws — APPROVED** [review]
LGTM. Left a couple of comments.

File: _ml-commons-plugin/extensibility/connectors.md [hunk 6]
```diff
@@ -76,39 +75,16 @@ If successful, the connector API responds with the `connector_id` for the connec
 }
 ```
 
+With the returned `connector_id` we can register a model that utilizing that connector.
```
Comment (natebower) [inline _ml-commons-plugin/extensibility/connectors.md:78]:
```suggestion
With the returned `connector_id` we can register a model that uses that connector:
```

File: _ml-commons-plugin/index.md [hunk 7]
```diff
@@ -17,18 +17,10 @@ Models [trained]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#training-the
...
-
-## ML node
-
-To prevent your cluster from failing when running ML tasks, you configure a node with the `ml` node role. When configuring without the `data` node role, ML nodes will not store any shards and will calculate resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Give your node a custom name and define the node role as `ml`:
-
-```yml
-node.name: ml-node
-node.roles: [ ml ]
-```
+1. Ensure you've set the appropriate cluster settings outlined in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/).
```
Comment (natebower) [inline _ml-commons-plugin/index.md:22]:
```suggestion
1. Ensure that you've appropriately set the cluster settings described in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/). 
```

File: _ml-commons-plugin/index.md [hunk 8]
```diff
@@ -17,18 +17,10 @@ Models [trained]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api#training-the
...
-## ML node
-
-To prevent your cluster from failing when running ML tasks, you configure a node with the `ml` node role. When configuring without the `data` node role, ML nodes will not store any shards and will calculate resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Give your node a custom name and define the node role as `ml`:
-
-```yml
-node.name: ml-node
-node.roles: [ ml ]
-```
+1. Ensure you've set the appropriate cluster settings outlined in [Cluster Settings]({{site.url}}{{site.baseurl}}/ml-commons-plugin/cluster-settings/). 
+2. Set up model access described in [Model Access Control]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control/).
```
Comment (natebower) [inline _ml-commons-plugin/index.md:23]:
```suggestion
2. Set up model access as described in [Model Access Control]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control/). 
```

**Review by natebower — COMMENTED** [review]
@dtaivpp Just a few wording changes. Otherwise, LGTM!

**Comment by dtaivpp** [comment]
@natebower and @kolchfa-aws accepted all the changes, thank you for reviewing!

**Comment by hdhalter** [comment]
@dtaivpp - Thank you for your bias for action in cleaning up this documentation! It is much appreciated.

**Pull request merged**